internal/commands/inspect_extension.go

```diff
@@ -0,0 +1,112 @@
+package commands
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+	"text/template"
+
+	"github.com/buildpacks/pack/pkg/buildpack"
+	"github.com/buildpacks/pack/pkg/client"
+)
+
+const inspectExtensionTemplate = `
+{{ .Location -}}:
+
+Stacks:
+{{- range $stackIndex, $stack := .Metadata.Stacks }}
+  ID: {{ $stack.ID }}
+    Mixins:
+  {{- if $.ListMixins }}
+    {{- if eq (len $stack.Mixins) 0 }}
+      (none)
+    {{- else }}
+      {{- range $mixinIndex, $mixin := $stack.Mixins }}
+      {{ $mixin }}
+      {{- end }}
+    {{- end }}
+  {{- else }}
+      (omitted)
+  {{- end }}
+{{- end }}
+
+Extensions:
```

Comment:
```suggestion
Extension:
```
(there is only ever one extension)